k8s/refs/broker_getter_func.go

```diff
@@ -0,0 +1,29 @@
+package refs
+
+import (
+	"github.com/deis/steward-framework/k8s/data"
+	"github.com/deis/steward-framework/k8s/restutil"
+	"k8s.io/client-go/pkg/api"
+	"k8s.io/client-go/rest"
+)
+
+// BrokerGetterFunc is the function that attempts to retrieve a broker at the given object ref
+type BrokerGetterFunc func(api.ObjectReference) (*data.Broker, error)
+
+// NewK8sBrokerGetterFunc returns a BrokerGetterFunc  backed by a real kubernetes client
+func NewK8sBrokerGetterFunc(restIface rest.Interface) BrokerGetterFunc {
+	return func(ref api.ObjectReference) (*data.Broker, error) {
+		ret := new(data.Broker)
+		url := restutil.AbsPath(
+			restutil.APIVersionBase,
+			restutil.APIVersion,
+			false,
+			ref.Namespace,
+			ref.Name,
+		)
```

Comment:
Good call. Fixed in b0a8dbfb105c9a8173f45e16da8e4cd8e6eff098, along with the other 2 types